webpack/test_setup.js

```diff
@@ -1,7 +1,3 @@
-import 'core-js/shim';
-import 'regenerator-runtime/runtime';
-
-import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-
-configure({ adapter: new Adapter() });
+/* This is for extending test setup from @theforeman 
```

Comment:
Can't we remove this file and make it optional?

Reply:
already done :)